classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    student.classroom = self
+    @students << student
```

Comment:
- Hey, in here, you need to push to the student array in this way below, kindly update it.
```
@students.push(student)
student.classroom = self
```

Reply:
Done!